NICE.Identity.Authorisation.WebAPI/Emails/inactive_account_removal.html

```diff
@@ -0,0 +1,119 @@
+	<center>
+		<table border="0" cellpadding="0" cellspacing="0" height="100%" width="100%" id="bodyTable" style="-ms-text-size-adjust: 100%; -webkit-text-size-adjust: 100%; border-collapse: collapse; height: 100%; margin: 0; mso-table-lspace: 0pt; mso-table-rspace: 0pt; padding: 0; width: 100%;">
+			<tr>
+				<td align="center" valign="top" style="-ms-text-size-adjust: 100%; -webkit-text-size-adjust: 100%; border-collapse: collapse; mso-line-height-rule: exactly; mso-table-lspace: 0pt; mso-table-rspace: 0pt;">
+					<br>
+					<table id="templateContainer" align="center" border="0" cellpadding="0" cellspacing="0" width="600" style="-ms-text-size-adjust: 100%; -webkit-text-size-adjust: 100%; border-collapse: collapse; mso-table-lspace: 0pt; mso-table-rspace: 0pt; width: 600px;">
+						<tr>
+							<td align="center" valign="top" style="-ms-text-size-adjust: 100%; -webkit-text-size-adjust: 100%; border-collapse: collapse; mso-line-height-rule: exactly; mso-table-lspace: 0pt; mso-table-rspace: 0pt;">
+								<table id="templateBody" border="0" cellpadding="20" cellspacing="0" width="100%" style="-ms-text-size-adjust: 100%; -webkit-text-size-adjust: 100%; background-color: #fff; border-collapse: collapse; mso-table-lspace: 0pt; mso-table-rspace: 0pt;">
+									<tr>
+										<td align="left" valign="top" style="-ms-text-size-adjust: 100%; -webkit-text-size-adjust: 100%; border-collapse: collapse; mso-line-height-rule: exactly; mso-table-lspace: 0pt; mso-table-rspace: 0pt;">
+											<img class="nice-logo" src="https://s3-eu-west-1.amazonaws.com/nice-identity-hostedpages/live/login/images/nicelogo.png" width="63%" style="-ms-interpolation-mode: bicubic; border: 0; height: auto; line-height: 100%; max-width: 380px; outline: none; text-decoration: none;">
+										</td>
+									</tr>
+									<tr>
+										<td align="left" valign="top" style="-ms-text-size-adjust: 100%; -webkit-text-size-adjust: 100%; border-collapse: collapse; mso-line-height-rule: exactly; mso-table-lspace: 0pt; mso-table-rspace: 0pt;">
+											<h1 style="color: #000; font-size: 22px; line-height: 100%; margin: 0; mso-line-height-rule: exactly;">Your dormant NICE Account has been deleted</h1>
+											<br />
+                                            Your NICE account has not been used for 3 years. Unfortunately, we have deleted your details in compliance with GDPR.
+											<br /><br />
+											If you want to re-register for a NICE account using this email address, you will have to re-submit your details on the <a href="https://www.nice.org.uk" target="_blank" style="-ms-text-size-adjust: 100%; -webkit-text-size-adjust: 100%; color: #005ea5;">NICE website</a>.
```

Comment:
looking at the template on the JIRA card, the url points to NICE accounts

Reply:
Well spotted again!